.github/workflows/nodejs.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [14.x, 16.x, 18.x]
```

Comment:
```suggestion
        node-version:
          - 14
          - 16
          - 18
          - 19
```

Reply:
I like it!

Reply:
In fact, why not make the node version dynamic? https://github.com/github/eslint-plugin-github/actions/runs/3802048520

Reply:
Up to you, but I think it's easier to understand the actual versions, and sync it with an `engines` value in https://github.com/github/eslint-plugin-github/blob/main/package.json